### PR TITLE
Added UI Components / Circle

### PR DESCRIPTION
**Pull request by saurabhalp**
### Added UI Components / AvatarCircle
- takes radius and image source as param
-returns circle component with an image inside.
- added separate files: 
   - `index.tsx`
   - `styles.ts`
   - added `react-native-svg` dependencies

### CODE REVIEW

**Comment by Aashirvaad-P** [comment]
Please change the naming of the folder:
src/UI/Components/AvatarCircle/ index.tsx and styles.d.ts

index.tsx for the component code 
styles.d.ts for styling

**Comment by saurabhalp** [comment]
Chnaged names and separated index.tsx and styles.ts

File: src/UI/Components/UI_Elements.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
Comment (Aashirvaad-P) [inline src/UI/Components/UI_Elements.tsx]: This file is not needed as the same code has been moved to AvatarCircle directory.

File: src/UI/Components/AvatarCircle/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    imageSource: ImageSourcePropType; // The image source, which can be a local image or a URL
+  }
+
+/**
+ * Circle component that displays an image within a circular frame.
+ * 
+ * @param {CircleProps} props - Props for the Circle component.
+ * @returns {JSX.Element} A circle component with an image inside.
+ */
+const AvatarCircle: React.FC<CircleProps> = ({ radius, imageSource }) => {
```
Comment (Aashirvaad-P) [inline src/UI/Components/AvatarCircle/index.tsx:8]: There is a small warning for doc: JSDoc types may be moved to TypeScript types. Do handle this.

File: src/UI/Components/AvatarCircle/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Image, ImageSourcePropType, View } from 'react-native';
+import Svg, { Circle as SvgCircle } from 'react-native-svg';
```
Comment (Aashirvaad-P) [inline src/UI/Components/AvatarCircle/index.tsx:2]: 'react-native-svg': This package has been used but not present in package.json. Just cross check once.

File: src/UI/Components/AvatarCircle/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+/**
+ * Circle component that displays an image within a circular frame.
+ * 
+ * @param {CircleProps} props - Props for the Circle component.
+ * @returns {JSX.Element} A circle component with an image inside.
+ */
+const AvatarCircle: React.FC<CircleProps> = ({ radius, imageSource }) => {
+  const diameter = radius * 2; // Calculate the diameter of the circle
```
Comment (Aashirvaad-P) [inline src/UI/Components/AvatarCircle/index.tsx]: Very basic calculation, no need for comment here.

**Comment by Aashirvaad-P** [comment]
Rest all works fine and usable Avatar working. Just fix the above comments.

**Comment by saurabhalp** [comment]
updated packages and removed javadocs

**Review by Aashirvaad-P — APPROVED** [review]
Works fine now

**Pull request merged**